### PR TITLE
added allowImageCache, imageType properties to draft-js-emoji-plugin config

### PR DESCRIPTION
**Pull request by Hapet95**

### CODE REVIEW

**Comment by nikgraf** [comment]
nice one @Hapet95 👌

can you also add an entry to the changelog and add it to the docs:

https://github.com/draft-js-plugins/draft-js-plugins/blob/master/draft-js-emoji-plugin/CHANGELOG.md

https://github.com/draft-js-plugins/draft-js-plugins/blob/master/docs/client/components/pages/Emoji/index.js#L137

**Comment by Hapet95** [comment]
thanks @nikgraf! The changelog & docs are done! And when are you planning to implement the new version?

File: docs/client/components/pages/Emoji/index.js [hunk 1]
```diff
@@ -136,7 +136,15 @@ export default class App extends Component {
           </div>
           <div className={styles.param}>
             <span className={styles.paramName}>imagePath</span>
-            <span>The Emojis are displayed using SVGs. The full path is constructed of multiple parts like this: {'`${imagePath}${unicode}.svg${cacheBustParam}`'}. The default imagePath is: '//cdn.jsdelivr.net/emojione/assets/svg/', but can be overwritten with this config.</span>
+            <span>The Emojis are displayed using SVGs. The full path is constructed of multiple parts like this: {'`${imagePath}${unicode}.${imageType}${cacheBustParam}`'}. The default imagePath is: '//cdn.jsdelivr.net/emojione/assets/svg/', but can be overwritten with this config.</span>
+          </div>
+          <div className={styles.param}>
+            <span className={styles.paramName}>imageType</span>
+            <span>The default imageType is: '.svg', but can be overwritten with this config.</span>
```
Comment (nikgraf) [inline docs/client/components/pages/Emoji/index.js]: Can you change it from `.svg` to `svg`? Is it correct that EmojiOne only provides `png` as an alternative option. If yes, can you mention it here?

**Comment by nikgraf** [comment]
Cool thx! I published a new beta today and probably do another on this weekend.

This is the roadmap for the final 2.0. Not too crazy anymore, but still a couple days work: https://github.com/draft-js-plugins/draft-js-plugins/issues/329

File: docs/client/components/pages/Emoji/index.js (hunk 1 above)
Comment (Hapet95) [inline docs/client/components/pages/Emoji/index.js]: Oh sorry) fixed. I not sure, because I'm using Twemoji2 with .png from CDN

**Comment by nikgraf** [comment]
@Hapet95 awesome change!

**Comment by Hapet95** [comment]
thx! so when you will realize the betta-8?)

**Comment by nikgraf** [comment]
today 😄

**Pull request merged**